### ISSUE TITLE
Fails to function when environment's encoding is cp932

### ISSUE THREAD

**Issue by mn7216** [Nixeld/MudaeAutoBot#9]
Having the Japanese language pack installed seems to result in the following error:
```
Logged in
File Not Found using Different Method
Using $settings found during search for channel XXXX
Exception ignored in thread started by: <bound method GatewayServer._response_loop of <discum.gateway.gateway.GatewayServer object at 0x000001FDFD2D2050>>
Traceback (most recent call last):
  File "C:\Users\XXXX\AppData\Roaming\Python\Python311\site-packages\discum\gateway\gateway.py", line 299, in _response_loop
    func(resp)
  File "C:\Users\XXXX\Documents\MudaeAutoBot\MudaeAutoBot.py", line 851, in on_message
    msg = get_server_settings(gid,matched_channel)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\XXXX\Documents\MudaeAutoBot\MudaeAutoBot.py", line 179, in get_server_settings
    pres_data.write(abcdef)
UnicodeEncodeError: 'cp932' codec can't encode character '\xb7' in position 53: illegal multibyte sequence
2024-03-03 16:46:32,507:Could not find channel XXXX, skipping waifu roll on this channel.
```

This also breaks functioning even if the bot successfully reads the channel (in my case I manually entered the channel settings in the XXXX.txt file in \channeldata). In this case, it will fail to recognize kakera & rolled characters.
I have not experienced a similar error with other python scripts.

**Comment by HANSSS9631** [comment]
I was not able to recreate the error with the windows japanese language pack, however i did a possible fix to your problem.
Tell me if it is working fine

**Comment by mn7216** [comment]
> I was not able to recreate the error with the windows japanese language pack, however i did a possible fix to your problem. Tell me if it is working fine

I've figured out that the cause is having cp932 as your preferred encoding in the environment (I'm not sure how that came about). I know it's at least related to the language pack because of the codec but now I'm not sure how or why python is seemingly using the cp932 codec just on my end.

I had tried the fix with changing the encoding to UTF-8 but that leads to the bot semi-functioning but failing to recognize kakera, rolled characters, & rolls left.
As a temporary fix, using [this](https://pypi.org/project/setdefaultencoding/) seems to fix the issue with:

```
import setdefaultencoding
setdefaultencoding.setdefaultencoding('utf-8')
```
